frontend/public/components/_catalog-item.scss

```diff
@@ -0,0 +1,57 @@
+$catalog-item-icon-size-lg: 40px;
```

Comment:
This was renamed from _cluster-service-class.scss. Only changes are to the filename and class names.